ibl_to_nwb/__init__.py

```diff
@@ -1,2 +0,0 @@
-from .AlyxToNWB import *
-from .NwbToOne import *
```

Comment:
`*` imports exposed at outer level of conversion pipelines often cause issues due to outdated dependencies that were intended to be pinned at a previous point in time

Removing them here